config/settings.py

```diff
@@ -10,6 +10,11 @@
 https://docs.djangoproject.com/en/3.2/ref/settings/
 """
 
+try:
+    from .local_settings import *
+except ImportError:
+    pass
```

Comment:
この記法を用いるときは以下のようにするとより利便性が向上できます
```suggestion
try:
    from .local_settings import *
    DEBUG = True
except ImportError:
    DEBUG = False
```